packages/graphql/src/translate/create-set-relationship-properties.ts

```diff
@@ -22,13 +22,64 @@ import type { Relationship } from "../classes";
 import mapToDbProperty from "../utils/map-to-db-property";
 import { addCallbackAndSetParam } from "./utils/callback-utils";
 import { matchMathField, mathDescriptorBuilder, buildMathStatements } from "./utils/math";
+import type { RelationshipAdapter } from "../schema-model/relationship/model-adapters/RelationshipAdapter";
 
 /*
     TODO - lets reuse this function for setting either node or rel properties.
            This was not reused due to the large differences between node fields
            - and relationship fields.
 */
```

Comment:
is this TODO still relevant after your change?

Reply:
moved it on top of the function that originally had this comment